docs/tests/integration/test_remoted/test_agent_communication/test_protocols_communication.md

```diff
@@ -46,5 +46,18 @@ a difference of 2 seconds.
 - **TCP,UDP and port 56000**
 - **UDP,TCP and port 56000**
 
+## Comments
+
+An important aspect to take into account is the time in which remoted makes a reload of the `client.keys` info.
```

Comment:
```suggestion
An important aspect to take into account is the time needed by wazuh-remoted to reload the `client.keys`.
```

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1101/commits/fd00b88abcd5a4dcf3af786dc0a7f86fb3d3a662